examples/external-data/src/mock-external-data-service/service.ts

```diff
@@ -89,13 +144,7 @@ export async function initializeExternalDataService(props: ServiceProps): Promis
 	/**
 	 * Register's the sender's URL to receive notifications when the external task-list data changes.
 	 *
-	 * Expected input data format:
-	 *
-	 * ```json
-	 * {
-	 *  url: string // The target URL to receive change notification
-	 * }
-	 * ```
+	 * Expected request body format: {@link RegisterWebhookRequest}
```

Comment:
Nit: We can probably just remove this line. Since the request parameter has this type, I think the constraint is discoverable enough without it.